src/features/locators/components/MessageForDuplicate.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { MouseEvent } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { checkDuplicates, evaluateCssSelector, evaluateXpath } from "../utils/utils";
+import { selectLocatorsByPageObject } from "../selectors/locatorsByPO.selectors";
+import { LocatorType } from "../../../common/types/common";
+import { setActiveSingle, setScrollToLocator } from "../locators.slice";
+import { JDNHash, ElementId } from "../types/locator.types";
+
+interface MessageForDuplicateProps {
+  locator: string;
+  resetAndCloseDialog: () => void;
```

Comment:
this prop can be just `closeDialog`, that's enough for this component and makes it independent of implementation of this callback

Reply:
fixed